Update ui_qt.cpp

davidhedlund opened the pull request:
## Description

Added the term "Quick Menu", and replaced "Big Picture menu" with "Big Picture menu (not to be confused with Steam's Big Picture mode)"

## Related Issues

Fixes:
* https://github.com/libretro/RetroArch/issues/18415

## Reviewers

@hizzlekizzle

davidhedlund committed:
Update ui_qt.cpp

Fixes:
* https://github.com/libretro/RetroArch/issues/18415

davidhedlund committed:
Update ui_qt.cpp

Align the descriptions of the F1 and F5 hotkeys with their exact sub-labels as specified in Settings -> Input -> Hotkeys

davidhedlund commented:
Perhaps it could be helpful to include the term 'window' for clarity when distinguishing between the two open windows: the **Desktop Menu window** and the **Main window**.

What do you think about that?

hizzlekizzle commented:
that seems pretty reasonable to me. As a user, would you be confused if it said that while you're running in fullscreen? (that may not even be possible, exactly, with the WIMP UI in that situation)

davidhedlund committed:
Update ui_qt.cpp

* Replace "Big Picture" with "main interface"
* add the term "window" in the description
  - The Main window must remain open for the Desktop Menu window to function; closing the Main window will also close the Desktop Menu window. So perhaps its better to include the term 'window' for clarity when distinguishing between the two open windows: the Desktop menu window and the Main interface window.

davidhedlund committed:
Update ui_qt.cpp

Replaced "This Desktop Menu window" with "The Desktop Menu window" to distinguish it from the Desktop Menu dialogue window containing that text.

davidhedlund committed:
Update ui_qt.cpp

Added "WIMP (Windows, Icons, Menus, Pointer)"

davidhedlund committed:
Update ui_qt.cpp

Added "QT" for easier identification (Help -> "About QT...")

davidhedlund committed:
Update ui_qt.cpp

Qt not QT

davidhedlund committed:
Update ui_qt.cpp

Wording ("Qt-based WIMP)"

davidhedlund commented:
> I'm not sure that adding the "QT" (should probably be stylized as "Qt" anyway) bit brings much to the table. It's more consistent with the 'help' menu, but I'm not sure that knowing the underlying toolkit name is actually beneficial/desirable for end-users.

* I have already changed it to "Qt"
* RetroArch generates retroarch_qt.cfg from Desktop Mode and retroarch.cfg from the main window. This provides users a clear way to distinguish Desktop Mode—in addition to WIMP terminology.

davidhedlund committed:
Update ui_qt.cpp

Clarify that these hotkey examples reflect the default key assignments, as users with custom mappings may find the examples don’t match their setup.

davidhedlund committed:
Update ui_qt.cpp

"Added 'View ->' before 'Settings -> Input -> Hotkeys' to show these settings are accessible from the Desktop Mode window.

davidhedlund commented:
Added 'View ->' before 'Settings -> Input -> Hotkeys' to show these settings are accessible from the Desktop Mode window.
* https://github.com/libretro/RetroArch/pull/18428/commits/f74920c3121fc7368ee18988583b55ef55dec3e0

davidhedlund committed:
Update ui_qt.cpp

minor

davidhedlund committed:
Update ui_qt.cpp

Appended '[tab]' to 'Hotkeys' for tabbed interface reference.

davidhedlund committed:
Update ui_qt.cpp

Clarified that settings are fully configurable from both Desktop Mode (View -> Settings...) and main interface (Main Menu -> Settings) with identical options, while noting certain actions/hotkeys only function in the main interface window.

SternXD commented:
For the big picture term thing I'd honestly prefer if a page was taken out of PCSX2s book:

User facing content: Big Picture Mode
Developer/source code content: FullscreenUI

davidhedlund committed:
Update ui_qt.cpp

Introduced the term "10-foot user interface" to describe the main window.

Reference: https://en.wikipedia.org/wiki/10-foot_user_interface

davidhedlund committed:
Update ui_qt.cpp

added the term Big Picture Mode

davidhedlund commented:
> For the big picture term thing I'd honestly prefer if a page was taken out of PCSX2s book:
> 
> User facing content: Big Picture Mode (BPM)

I added the term "Big Picture Mode" in addition:
* https://github.com/libretro/RetroArch/pull/18428/changes/77cf8b003d743500c75aa6bbf3ec011310ffb9c4

davidhedlund committed:
Update ui_qt.cpp

minor
